app/static/about.css

```diff
@@ -0,0 +1,56 @@
+body {
+    margin: 0;
```

Comment:
this is already set the the global.css, please connect it to your pages. 
(check for all those general settings)

Reply:
added!